fsutils/ipcfg/ipcfg.c

```diff
@@ -498,14 +504,16 @@ int ipcfg_read(FAR const char *netdev, FAR struct ipcfg_s *ipcfg)
  ****************************************************************************/
 
 #ifdef CONFIG_IPCFG_WRITABLE
-int ipcfg_write(FAR const char *netdev, FAR const struct ipcfg_s *ipcfg)
```

Comment:
After Line 498 needs the af param doc added

Reply:
Fixed